docs/app/pages/components/components-sections/timeline/timeline/timeline.component.ts

```diff
@@ -0,0 +1,84 @@
+import { Component } from '@angular/core';
+import { DocumentationSectionComponent } from '../../../../../decorators/documentation-section-component';
+import { BaseDocumentationSection } from '../../../../../components/base-documentation-section/base-documentation-section';
+import 'chance';
+
+@Component({
+    selector: 'uxd-components-timeline',
+    templateUrl: './timeline.component.html'
+})
+@DocumentationSectionComponent('ComponentsTimelineComponent')
+export class ComponentsTimelineComponent extends BaseDocumentationSection {
+
+    constructor() {
+        super(require.context('./snippets/', false, /\.(html|css|js|ts)$/));
+    }
+
+    _developer = chance.name();
+
+    tickets = [{
```

Comment:
Could you create an interface for these items to ensure they always conform to the expected structure

Reply:
Added TimelineElement interface to define structure of TimelineEvent. The 'shortDate' value only is required. This was agreed earlier with Donna. It was also agreed that the 'date' property would be exposed as a Date.